### PR TITLE
ppsspp 1.1.0

### PR DESCRIPTION
**Pull request by tomyun**
- version bump
- add new deps: `glew`, `snappy`
- use `DUSE_SYSTEM_FFMPEG=ON` instead of `inreplace`
- add `test` to bypass audit warning

### CODE REVIEW

File: ppsspp.rb [hunk 1]
```diff
@@ -34,4 +30,8 @@ def install
       bin.write_exec_script "#{prefix}/PPSSPPSDL.app/Contents/MacOS/PPSSPPSDL"
     end
   end
+
+  test do
+    File.executable? "#{prefix}/PPSSPPSDL.app/Contents/MacOS/PPSSPPSDL"
```
Comment (bfontaine) [inline ppsspp.rb]: Please add a test for the actual functionality.

Comment (tomyun) [inline ppsspp.rb]: I'd love to make one, but Mac version of ppsspp doesn't support any command line arguments, so couldn't come up with a proper one. Do you think we'd better remove this block then?

Comment (bfontaine) [inline ppsspp.rb]: Yeah, it’s better to remove it.

Comment (tomyun) [inline ppsspp.rb]: It's now removed.

**Comment by dunn** [comment]
Merged in 5f97986c2f4e936a1ef701f0db474e9415710f3c, thanks again!

**Pull request closed without merging**